app/ui/src/main/java/com/fsck/k9/fragment/MessageListAdapter.kt

```diff
@@ -17,35 +18,18 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.CursorAdapter
 import android.widget.TextView
-
+import androidx.core.view.isVisible
 import com.fsck.k9.Account
-import com.fsck.k9.ui.R
 import com.fsck.k9.Preferences
 import com.fsck.k9.contacts.ContactPictureLoader
 import com.fsck.k9.controller.MessageReference
+import com.fsck.k9.fragment.MLFProjectionInfo.*
```

Comment:
Code style from the `settings.jar` and Kotlin still defaults  to wildcard. Do we want this import style in Kotlin?

Reply:
The `settings.jar` is pretty old. I believe it doesn't contain any Kotlin-specific settings. So the IDE's defaults are used.

Reply:
Done